src/components/Shared/ProfilerInfo/ProfilerInfoComponent.tsx

```diff
@@ -6,18 +6,16 @@ interface ProfilerInfoComponentProps {
 }
 
 export const ProfilerInfoComponent: React.FC<ProfilerInfoComponentProps> = (props) => {
-    let infoTags = null;
-    if (props.info && props.info.length > 0) {
-        infoTags = props.info.map((infoString, index) => <tr key={index}><td><pre>{infoString}</pre></td></tr>);
+    let infoString = "";
+    if (props.info?.length > 0) {
+        props.info.forEach((info, index) => {
```

Comment:
can't we just use `props.info?.forEach`?

Reply:
Refined.